### DISCUSSION TITLE
Cheat sheet HTML

### DISCUSSION THREAD

**Discussion by TherealVL** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#92] [Ideas]
Hola! Encontré esta cheat sheet de HTML muy pertinente a lo que veremos en la semana. 

Les serviría para conocer un poco más acerca de las etiquetas HTML.

Los créditos vienen en la imagen. 

PD: Por favor ya no posteen en la 1 de la madrugada, necesitamos dormir. La pregunta o la respuesta puede esperar en la mañana. 🥺👌

![Screenshot_2022-02-23-15-54-19-927.jpg](https://user-images.githubusercontent.com/84485246/155416423-b63c438e-ee8e-4a13-9110-9d22e4d25e77.jpg)
![Screenshot_2022-02-23-15-55-15-878.jpg](https://user-images.githubusercontent.com/84485246/155416507-352a0db8-b56d-4685-aac0-22ff4a6154c9.jpg)

**Comment by GloriaCoralCerecedo** [comment]
Muchas gracias, y en ese caso puedes desactivar las notificaciones para que no haya problema

**Comment by alanglez45** [comment]
You can disable notifications, papito.
